### ISSUE TITLE
Change screen corner radius of of home/lockscreen/apps/appswitcher cards

### ISSUE THREAD

**Issue by Ziusun** [MTACS/Lynx2#404]
Hey, 

title says it all :). From what I read this was already part of Lynx. Did it get removed for some reason?

**Comment by MTACS** [comment]
Most likely the methods to apply it were changed since iOS 14 and I never re-added it back, it will be included in the next update

**Comment by Ziusun** [comment]
I saw you added an option for the app switcher in the newest update, that is very welcome 👍🏻. Is that option still planned for the regular screen?

**Comment by iamn0tdev** [comment]
@Ziusun Yes. The changes added in 2.5.1 are only applied to the app swicther and not to the whole system like Reachability and apps animations. Hopefully @MTACS will be able to add it system-wide in a future update.
